### PR TITLE
Docker supported for development and testing only

### PR DESCRIPTION
**Pull request by bmansheim**

### CODE REVIEW

**Comment by bmansheim** [comment]
Staging site: https://docs.redislabs.com/staging/docker-for-dev/

File: content/rs/administering/designing-production/supported-platforms.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ systems, cloud environments, and virtual environments.
 | RHEL/CentOS 7 | 7.0, 7.1, 7.2, 7.3, 7.4, 7.5, 7.6<br>Requires at least "Minimal Install" configuration. For additional considerations, refer to [CentOS / RHEL 7 firewall configuration]({{< relref "/rs/installing-upgrading/configuring/centos-rhel-7-firewall.md" >}}). |
 | Oracle Linux | 6.7, 6.8, 6.9; 7.0, 7.1, 7.2, 7.3, 7.4, 7.5 |
 | Amazon Linux | All 64-bit Versions |
-| Docker | At this time, official Redis Enterprise Software Docker images are Linux-based images and therefore are only certified for production use on a Linux host. All other OS hosts are certified for Development and Testing only. |
+| Docker | At this time, official Redis Enterprise Software Docker images are Linux-based images and are certified for Development and Testing only. |
```
Comment (Paz-Yanover) [inline content/rs/administering/designing-production/supported-platforms.md]: I would say something like:
"Redis Enterprise Software Docker images are certified for Development and Testing only. "

File: content/rs/administering/database-operations/create-crdb.md [hunk 2]
```diff
@@ -120,13 +120,7 @@ replication to for the global CRDB.
...
 redis-cli is a simple command-line tool to interact with redis database.
 
-1. To switch your context into the RS container of node 1 in cluster 1, run:
-
-    ```src
-    docker exec -it rp1_node1 bash
-    ```
-
-1. To use redis-cli on port 12000, run:
+1. To use redis-cli on port 12000 from the node 1 terminal, run:
```
[the remark this answers is not in the document]
Comment (bmansheim) [reply content/rs/administering/database-operations/create-crdb.md]: We don't mention Docker in the setup section so it does not belong here either. Also, this article is intended for production environments.

File: content/rs/administering/database-operations/create-crdb.md [hunk 3]
```diff
@@ -147,16 +141,12 @@ redis-cli is a simple command-line tool to interact with redis database.
     "123"
     ```
 
-1. Enter `exit` to exit the redis-cli context and enter `exit` again to exit the
-   RS container of node 1 in cluster 1.
-1. To see that the key replicated to cluster 2, repeat the steps to switch your
-   context into the RS container of node 1 in cluster 2, run the redis-cli and
+1. Enter the terminal of node 1 in cluster 2, run the redis-cli, and
```
Comment (bmansheim) [inline content/rs/administering/database-operations/create-crdb.md]: We don't mention Docker in the setup section so it does not belong here either. Also, this article is intended for production environments.

**Pull request merged**